docs/modules/reference/pages/configuration/sentinel-features.adoc

```diff
@@ -327,3 +327,81 @@ admin@sentinel> config:update
 
 Please note, that the file `/opt/sentinel/etc/junos-telemetry-interface.groovy` must also be provided manually,
 e.g. by manually copying it over from _{page-component-title}_.
+
+== Configuring Sentinel via confd
+
+=== Mounting
+When starting the Sentinel container, mount a yaml file to the following path `/opt/sentinel/sentinel-config.yaml`.
+
+Any configuration provided to confd will overwrite configuration specified as environment variables. Direct overlay of
+specific configuration files will overwrite the corresponding config provided by confd.
+
+=== Contents
+The following describes the keys that can be specified in `sentinel-config.yaml` to configure the Sentinel via confd.
+
+==== Sentinel Controller Config
+[source, yaml]
+----
+broker-url: "<broker url>"
+id: "<id>"
+location: "<location>"
+----
+Config specified will be written to `etc/org.opennms.sentinel.controller.cfg`.
+
+==== User/Password
+Supplying the broker username/password via yaml file for configuration via confd is not supported.
+
+==== Sentinel Elasticsearch Config
```

Comment:
```suggestion
==== Sentinel Elasticsearch config
```

Reply:
By the way, I don't like our heading capitalization rules, as we end up with things like this where most of the heading is capitalized and one word isn't. Sigh.